lib/input_validator.rb

```diff
@@ -0,0 +1,17 @@
+module TicTacToe 
+  class InputValidator
+
+    def is_size_valid?(size)
+      size <= 10 && size > 2
+    end
+
+    def is_marker_valid?(marker)
+      marker.capitalize != "X" && marker.length == 1 ? true : false
```

Comment:
This could be moved down into a private method - it's not immediately clear what the significance of the marker equalling 1 is.